### ISSUE TITLE
Add activation parameter to BatchNormalization

### ISSUE THREAD

**Issue by eli-osherovich** [keras-team/keras#14849]
### Describe the problem
This is a feature request: add `activation` parameter to the `BatchNormalization` layer. 

This layer is often followed by an activation function, e.g., `ReLU`, hence it would be convenient to provide it as a parameter similar to the `Dense` layer.

**Comment by rmothukuru** [comment]
@eli-osherovich,
Can you please provide more details and a use case where this Feature is applicable? Thanks!

**Comment by eli-osherovich** [comment]
@rmothukuru 
Most modern architectures use BN followed by an activation layer (typically ReLU). It would be nice (and more aligned with other layers, e.g., Dense, Conv) to have activation parameter directly within the BN layers. The code will also be more concise.

**Comment by qlzh727** [comment]
@fchollet: leave this to Francois as API owner for further comment.

**Comment by SuryanarayanaY** [comment]
Hello, Thank you for reporting an issue.

We're currently in the process of migrating the new Keras 3 code base from keras-team/keras-core to keras-team/keras.
Consequently, This issue may not be relevant to the Keras 3 code base . After the migration is successfully completed, feel free to reopen this Issue at keras-team/keras if you believe it remains relevant to the Keras 3 code base. If instead this Issue is a bug or security issue in legacy tf.keras, you can instead report a new issue at [keras-team/tf-keras](https://github.com/keras-team/tf-keras/issues), which hosts the TensorFlow-only, legacy version of Keras.

To know more about Keras 3, please read https://keras.io/keras_core/announcement/
